src/app/routes/article/getInitialData/index.ts

```diff
@@ -23,6 +25,11 @@ const getEnvironment = (pathname: string) => {
   if (pathname.includes('renderer_env=live')) {
     return 'live';
   }
+  if (pathname.includes('renderer_env=caf')) {
+    return process.env.SIMORGH_APP_ENV !== 'local'
+      ? process.env.SIMORGH_APP_ENV
+      : 'live';
+  }
```

Comment:
If `renderer_env=caf` is supplied, it will set the environment to whatever environment Simorgh is running in **if its not running locally**, in which case it will default to `live`.

This is to set the `ctx-service-env` header for fetching from the BFF. Seemed to make more sense to test against Live assets since the metadata and page data will be much more accurate for comparison. 

You will need to manually set the BFF url to point to your locally running instance of FABL if you want to test FABL locally.